cocotb/share/makefiles/Makefile.pylib.Msys

```diff
@@ -47,3 +47,11 @@ ifeq ($(PYTHON_LIBDIR),None)
 endif
 
 PYTHON_INCLUDEDIR := $(shell $(PYTHON_BIN) -c 'import distutils.sysconfig; print( distutils.sysconfig.get_python_inc() )')
+
+# For Tcl script purposes, generate source file paths in the form of C:/User/...
```

Comment:
If this is for tcl scripts, should we perform this transformation at the point we emit tcl code?

Reply:
It is 2 files (quest and aldec). Where it should go? It should help with any input path construction.

Reply:
Perhaps define a `to_tcl_path` function using `define` based on Windows vs posix, and call that function from both makefiles?

Reply:
Does this idea work?

Reply:
It should work. Can try. Byt all this extra WPWD in user makefiles is also working if applied consistently.  Mether of taste.